vector/src/main/res/values/strings.xml

```diff
@@ -1112,6 +1112,10 @@
     <string name="settings_notification_advanced">Advanced Notification Settings</string>
     <string name="settings_notification_by_event">Notification importance by event</string>
 
+    <string name="settings_notification_emails_category">Email notification</string>
+    <string name="settings_notification_emails_no_emails">To receive email with notification, please associate an email to your Matrix account</string>
+    <string name="settings_notification_emails_enable_for_email">Enable email notifications for %s</string>
```

Comment:
Can you add a XML comment for translators, just above this line to tell them that the placeholder will be replaced by an email? There are other comment like that in this file. Thanks.

Reply:
1d4c909b50b5b9f4ed8656ee71b340c3f7310cc3